python_ta/__init__.py

```diff
@@ -166,6 +166,24 @@ def _load_config(linter, config_location):
     linter.load_config_file()
 
 
+def _load_messages_config(path: str, default_path: str) -> dict:
+    """Given path (potentially) specified by user and default default_path
+    of messages config file, merge the config files."""
+    merge_into = toml.load(default_path)
+    merge_from = toml.load(path)
```

Comment:
@peelstnac I thought about this a bit more carefully and have a couple more things I'd like you to update here. (And, investigating a runtime issue that @harshkpatel reported.)

1. If `path == default_path`, we should just return the default configuration, no need to reload the same file and merge.
2. If `path` cannot be found (because of an error in the configuration file), please output a warning message (similar to other PythonTA warnings) and return the default config. Right now `toml.load` will raise an error and halt execution altogether.
3. In the default `.pylintrc` file, please *comment out* the config value for the messages config (so that no value is used by default), and change the example value to just `messages.toml` or something similar, without the parent folders. The relative path structure is a bit misleading.